contracts/cairo/deploy.sh

```diff
@@ -32,13 +30,14 @@ echo -e "- ${PINK}[ETH] Market Maker: $MM_ETHEREUM_WALLET${COLOR_RESET}"
 
 echo -e "${GREEN}\n=> [SN] Deploy Escrow${COLOR_RESET}"
 ESCROW_CONTRACT_ADDRESS=$(starkli deploy \
-  --account $STARKNET_ACCOUNT --keystore $STARKNET_KEYSTORE \
+  --account $STARKNET_ACCOUNT --keystore $STARKNET_KEYSTORE --rpc $SN_RPC_URL \
   --watch $ESCROW_CLASS_HASH \
+    $SN_ESCROW_OWNER \
```

Comment:
this param is not explaines in the readme. What is it for and where do i gets its expected value from?

Reply:
If it is the pubkey of the owner of the smart contract, we could consider having deploy.sh writing this .env var automatically from the keystore being used

Reply:
What if we don't want the deployer to be the owner? Wouldn't this limit it? While automation can make things easier for the user, in this case, it seems to make it less flexible